### PR TITLE
contact form

### PR DESCRIPTION
**Pull request by godybuoy**
Add a contact form to finish first mobile setup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+      <img src="image/contact form shapes-mobile@2x.svg" alt="">
+      <p>If you have an application you are interested
+         in developing, a feature that you need built or a project
+          that needs coding. I’d
+         love to help with it </p>
+         <form class="form-start" action="">
+          <input type="name" type="text" name="name" class="name" placeholder="write your name">
+          <input type="email" type="email" name="email" class="email" placeholder="write your email">
+          <input type="message" type="text" name="message" class="message" placeholder="write your message">
+          <button class="start">get in touch</button>
```
Comment (mugishasam123) [inline index.html:206]: - [ ] Kindly try to stick to the figma design as much as possible to make your project looks good. try to leverage padding colors etc.

Your design:
image

Figma design:
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+      <h1>contact me</h1>
+      <img src="image/contact form shapes-mobile@2x.svg" alt="">
+      <p>If you have an application you are interested
+         in developing, a feature that you need built or a project
+          that needs coding. I’d
+         love to help with it </p>
+         <form class="form-start" action="">
+          <input type="name" type="text" name="name" class="name" placeholder="write your name">
+          <input type="email" type="email" name="email" class="email" placeholder="write your email">
+          <input type="message" type="text" name="message" class="message" placeholder="write your message">
```
Comment (mugishasam123) [inline index.html:205]: - [ ] Please try to implement all form features as it is stated in the project guidelines to make your project look professionally.(form validations like length,required etc)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  </section>
+  <section class="contact">
+    <article class="contact_footer">
+      <h1>contact me</h1>
+      <img src="image/contact form shapes-mobile@2x.svg" alt="">
+      <p>If you have an application you are interested
+         in developing, a feature that you need built or a project
+          that needs coding. I’d
+         love to help with it </p>
+         <form class="form-start" action="">
```
Comment (mugishasam123) [inline index.html:202]: - [ ] Please try to make your form functional by integrating with form spree service as it was stated in the project guidelines.

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @godybuoy  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive PR title 👍 
- No linter errors 🥇 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by godybuoy** [comment]
I followed form spree services and implemented all form features

**Review by emmiiorji — COMMENTED** [review]
Hi @godybuoy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid

You have submitted a pull request that has already been merged. It is required to not merge a pull request until it's approved, kindly undo the merge.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by godybuoy** [comment]
@emmiiorji Hi! Thank you for your help. I made a merge and then submitted. should I unmerge or open a new pool request.

**Comment by joskalenda** [comment]
> @emmiiorji Hi! Thank you for your help. I made a merge and then submitted. should I unmerge or open a new pool request.

Hey @godybuoy, I got your recent pull request submission, has said by this previous reviewer, you should not merge a pull request before it is approved, in case you do it, as you did with this one, please open a new one and add the link to the previous pull request in the newly created. << Please keep this in mind >>

⚠️  I will invalidate your new pull request because it does not have a descriptive pull request summary, please update it, and do not forget to add the merged pull request link for your future submission.

**Pull request merged**